notebooks/scripts/example_study.py

```
@@ -0,0 +1,106 @@
+    print('number of patients', mace_df.PID.nunique(), end=' ')
+    print('number of events', mace_df.shape[0],)
+    mace_pids.extend(mace_df.PID.unique())
+
+# %%
+diag.CONCEPT.value_counts()[25:40]
+
+# %% [markdown]
+# ## Examine statin codes
+
+# %%
+med
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)